components/moon.tsx

```diff
@@ -1,16 +1,21 @@
 import type { moon } from '@/lib/types';
 import { useEffect, useState } from 'react';
+import Loading from './loading';
+import Bee from './bee';
 
 interface MoonProps {
   planetName: string | string[] | undefined;
 }
 
 const Moons = (props: MoonProps) => {
   const [moonData, setMoonData] = useState<moon[]>();
+  const [loading, setLoading] = useState<boolean>(true);
+  const [showMoon, setShowMoon] = useState<string>('');
   const { planetName } = props;
 
   useEffect(() => {
     const fetchMoonData = async () => {
+      setLoading(true);
```

Comment:
Because you're initializing loading to true on line 12, I don't believe we need a loading animation here.

Reply:
Gotcha, removed it here as well as on the [planetName].tsx file 👍 

![image](https://user-images.githubusercontent.com/109567499/229324780-fa5b5803-f158-44fa-9121-73a482a09f13.png)